RNA-seq/02-gastric_cancer_tximeta.Rmd

```diff
@@ -156,16 +156,16 @@ We more specifically have a `RangedSummarizedExperiment` which is a more specifi
 
 This figure is from this handy vignette about [`SummarizedExperiment` objects](https://www.bioconductor.org/packages/devel/bioc/vignettes/SummarizedExperiment/inst/doc/SummarizedExperiment.html).
 
-As shown in the diagram, we can use some of these functions to extract data from our `SummarizedExperiment` object. 
-`rowData()` on our object shows all the gene information that `tximeta` set up!
+As shown in the diagram, we can use some of the functions provided by the `SummarizedExperiment` package to extract data from our `gene_summarized` object. 
+For example, calling `rowData()` on our object shows all the gene information that `tximeta` set up!
 
 ```{r}
 # rowData() shows us our gene annotation
 rowData(gene_summarized)
 ```
 
-The `assay` slot in `SummarizedExperiment`s hold the experiment data, a gene x sample matrix. 
-In this case, it has our gene-level abundance information. 
+The `assay` slot in `SummarizedExperiment`s holds data from the experiment. 
+In this case, it will include our gene-level abundance information stored as a gene x sample matrix. 
```

Comment:
Modifying to not create confusion with `abundance`
```suggestion
In this case, it will include our gene-level expression information stored as a gene x sample matrix. 
```